modules/systemd.nix

```diff
@@ -1,18 +1,24 @@
-{ config, lib, pkgs, ... }:
+{ config, lib, pkgs, utils, ... }:
 
+with utils;
```

Comment:
Which function from `utils` is used? I can't find any.

Reply:
Sorry, left over from removed code.